CHANGELOG.md

```diff
@@ -1,7 +1,23 @@
+
 # Changelog
 
 ## v1.19.0 - TBD
 
+### Storage
+
+* fix: consistent computation of XML vs. JSON endpoints
```

Comment:
This is an interesting pattern you're doing here. Maybe we should do this more often when making a change that requires a nuanced description in the changelog. I like this. Thanks.